pom.xml

```diff
@@ -519,6 +520,11 @@
                 <artifactId>postgresql</artifactId>
                 <version>${postgresql.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Please add license

Reply:
The license is not a protocol type allowed by Apache. If you want to use it, it can be provided in the form of Provided

Reply:
ok,i do it latter.